yax/core/src/main/scala/doobie/hi/resultset.scala

```diff
@@ -206,7 +206,7 @@ object resultset {
       val b = Vector.newBuilder[A]
       while (n > 0 && rs.next) {
```

Comment:
This is by design. I guess we could throw an exception since it doesn't really make any sense but it doesn't bother me as-is.